Tentacle/Scripts/configure-tentacle.ps1

```diff
@@ -9,8 +9,8 @@ $TargetEnvironment = $env:TargetEnvironment;
 $TargetRole = $env:TargetRole;
 $TargetName=$env:TargetName;
 $ListeningPort=$env:ListeningPort;
-$PublicHostNameConfiguration=$env:PublicHostNameConfiguration
-
+$PublicHostNameConfiguration=$env:PublicHostNameConfiguration;
+$ComsStyle=$env:ComsStyle;
```

Comment:
I think this would be better as `CommsStyle` to match the parameter further down.